[rospy.client][INFO] 2020-12-26 05:36:55,522: init_node, name[/train_5011_1608961015494], pid[5011]
[xmlrpc][INFO] 2020-12-26 05:36:55,522: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-12-26 05:36:55,523: Started XML-RPC server [http://192.168.0.191:41947/]
[rospy.init][INFO] 2020-12-26 05:36:55,523: ROS Slave URI: [http://192.168.0.191:41947/]
[rospy.impl.masterslave][INFO] 2020-12-26 05:36:55,523: _ready: http://192.168.0.191:41947/
[rospy.registration][INFO] 2020-12-26 05:36:55,525: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-12-26 05:36:55,525: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-12-26 05:36:55,624: registered with master
[rospy.rosout][INFO] 2020-12-26 05:36:55,624: initializing /rosout core topic
[rospy.rosout][INFO] 2020-12-26 05:36:55,628: connected to core topic /rosout
[rospy.simtime][INFO] 2020-12-26 05:36:55,630: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-12-26 05:36:55,645: topic[/bev/temporal_bev_image] adding connection to [http://192.168.0.201:39549/], count 0
[rospy.internal][INFO] 2020-12-26 05:36:55,659: topic[/scan] adding connection to [http://192.168.0.201:40197/], count 0
[rospy.internal][INFO] 2020-12-26 05:36:55,661: topic[/scan] adding connection to [http://192.168.0.201:37833/], count 1
[rospy.internal][INFO] 2020-12-26 05:36:55,664: topic[/start_goal_points] adding connection to [http://192.168.0.201:40835/], count 0
[rospy.internal][INFO] 2020-12-26 05:36:55,665: topic[/is_start_episode] adding connection to [http://192.168.0.201:42987/], count 0
[rospy.internal][INFO] 2020-12-26 05:36:55,667: topic[/odom] adding connection to [http://192.168.0.201:40197/], count 0
[rospy.internal][INFO] 2020-12-26 05:36:55,854: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-12-26 05:36:55,866: topic[/episode_watcher/is_finish_episode] adding connection to [/episode_manager], count 0
[rospy.internal][INFO] 2020-12-26 05:36:55,869: topic[/cmd_vel] adding connection to [/gazebo], count 0
[rospy.core][INFO] 2020-12-26 05:37:00,850: signal_shutdown [atexit]
[rospy.internal][INFO] 2020-12-26 05:37:00,855: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-12-26 05:37:00,856: topic[/cmd_vel] removing connection to /gazebo
[rospy.internal][INFO] 2020-12-26 05:37:00,856: topic[/episode_watcher/is_finish_episode] removing connection to /episode_manager
[rospy.internal][INFO] 2020-12-26 05:37:00,856: topic[/bev/temporal_bev_image] removing connection to http://192.168.0.201:39549/
[rospy.internal][INFO] 2020-12-26 05:37:00,857: topic[/odom] removing connection to http://192.168.0.201:40197/
[rospy.internal][INFO] 2020-12-26 05:37:00,857: topic[/start_goal_points] removing connection to http://192.168.0.201:40835/
[rospy.internal][INFO] 2020-12-26 05:37:00,858: topic[/scan] removing connection to http://192.168.0.201:40197/
[rospy.internal][INFO] 2020-12-26 05:37:00,858: topic[/scan] removing connection to http://192.168.0.201:37833/
[rospy.internal][INFO] 2020-12-26 05:37:00,859: topic[/is_start_episode] removing connection to http://192.168.0.201:42987/
[rospy.impl.masterslave][INFO] 2020-12-26 05:37:00,860: atexit
[rospy.internal][WARNING] 2020-12-26 05:37:00,860: Unknown error initiating TCP/IP socket to 192.168.0.201:56385 (http://192.168.0.201:40197/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2020-12-26 05:37:00,861: Unknown error initiating TCP/IP socket to 192.168.0.201:56385 (http://192.168.0.201:40197/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

